edit/base.js

```diff
@@ -48,6 +63,19 @@ const editor = Object.assign(EventEmitter(), {
       customName || name || t('styleMissingName')
     } - Stylus`; // the suffix enables external utilities to process our windows e.g. pin on top
   },
+};
+
+editor.on('styleUpdated', (newStyle, reason) => {
+  if (reason === 'config') {
+    delete newStyle.sourceCode;
+    delete newStyle.sections;
+    delete newStyle.name;
+    delete newStyle.enabled;
+    Object.assign(editor.style, newStyle);
```

Comment:
IIRC, the editor doesn't always use `editor.style` so I implemented them separately and added an event listener.

Reply:
It's assigned in base.js::loadStyle which runs before edit.js can emit `styleUpdated`.